sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuExpressions.scala

```diff
@@ -148,6 +148,10 @@ trait CudfUnaryExpression extends GpuUnaryExpression {
   override def doColumnar(input: GpuColumnVector): ColumnVector = input.getBase.unaryOp(unaryOp)
 }
 
+trait GpuRoundBase extends GpuBinaryExpression {
```

Comment:
Unless we are going to include more code that uses `roundMode` in here I don't see any reason for having this trait.